modules/ossm-excluding-namespaces-from-cluster-wide-mesh-cli.adoc

```diff
@@ -25,7 +25,12 @@ $ oc -n istio-system edit smcp <name> <1>
 ----
 <1> `<name>` represents the name of the `ServiceMeshControlPlane` resource.
 
-. Modify the YAML file so that the `spec.discoverySelectors` field of the `ServiceMeshMemberRoll` resource includes the discovery selector. The following example uses `istio-discovery: enabled`:
+. Modify the YAML file so that the `spec.discoverySelectors` field of the `ServiceMeshMemberRoll` resource includes the discovery selector. In the following example, Istiod discovers any namespace labeled `istio-discovery=enabled` or any namespace with name `bookinfo`, `httpbin` or `istio-system`.
```

Comment:
Maybe we could change this to `istio-discovery: enabled` to remain consistent with the rest of the doc.

Reply:
done